pkg/controller/importconfig/cluster_info.go

```diff
@@ -482,3 +486,62 @@ func getBootstrapSAName(clusterName string) string {
 	}
 	return bootstrapSAName
 }
+
+func filesToBytes(files ...string) []byte {
```

Comment:
is this just the same as filesToTemplateBytes(files, nil)?

Reply:
Very true, fixed.